mercure.rst

```diff
@@ -159,20 +159,26 @@ service, including controllers::
     namespace App\Controller;
 
     use Symfony\Component\HttpFoundation\Response;
-    use Symfony\Component\Mercure\PublisherInterface;
+    use Symfony\Component\Mercure\HubInterface;
     use Symfony\Component\Mercure\Update;
 
     class PublishController
     {
-        public function __invoke(PublisherInterface $publisher): Response
+        private $hub;
+
+        public function __construct(HubInterface $hub)
```

Comment:
I suggest to stick to controller argument injection for simplicity and consistency with the rest of the docs.